open_feature/open_feature_client.py

```diff
@@ -149,34 +149,65 @@ def evaluate_flag_details(
         flag_type: FlagType,
         key: str,
         default_value: typing.Any,
-        evaluation_context: typing.Any = None,
+        evaluation_context: EvaluationContext = None,
         flag_evaluation_options: typing.Any = None,
     ) -> FlagEvaluationDetails:
+        if evaluation_context is None:
+            evaluation_context = EvaluationContext()
         try:
-            args = (
+            # The merge below will eventually pull in Evaluation contexts from Hooks
+            invocation_context = EvaluationContext()
+            invocation_context.merge(ctx1=EvaluationContext(), ctx2=evaluation_context)
```

Comment:
Related to the above - I'm not sure why we're creating 2 new EvaluationContext objects?

Reply:
Yes it' unnecessary, I'll make the change

Reply:
Seems like these changes were dropped entirely, are they going to be in a separate PR?